### PR TITLE
Ydmeta.adopt.to.gitpod

### PR DESCRIPTION
**Pull request by ydmeta**
Add poetry environment for dev
add pyinstaller build
add poetry build
organize task
update docs
@djfedos @lioryah 
please review and approve

### CODE REVIEW

File: _infra/setup-dev-station/install.os.tool.yq.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Habits
```
Comment (djfedos) [inline _infra/setup-dev-station/install.os.tool.yq.md:2]: Lines 2, 3 and 4 look wierd like they're from example. Are they OK?

File: Taskfile.full.yml [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    desc: run command within the venv
+    cmds:
+      - |-
+        bash --login -c '
+        conda activate {{.const_venv}}
+        {{.cmd}}
+        '
+
+  verify-request:
+    desc: check syntax, staling, etc
```
Comment (djfedos) [inline Taskfile.full.yml:133]: @ydmeta, you mean styling?

**Review by djfedos — APPROVED** [review]
Looks OK

File: Taskfile.full.yml [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+      - task: stam
+      - echo task build-pyinst-pack
+
+  stam:
+    desc: _
+    cmds:
+      - echo one
+      - task: pyfull
+
+  test-them: test them
```
Comment (djfedos) [inline Taskfile.full.yml:163]: @ydmeta, what does this command do?

**Review by djfedos — APPROVED** [review]
Looks OK

**Review by lioryah — APPROVED** [review]
Reviewed the changes. 

what is  act_init_venv_nb.py file?

File: _infra/setup-dev-station/install.os.tool.yq.md (hunk 1 above)
Comment (ydmeta) [reply _infra/setup-dev-station/install.os.tool.yq.md:2]: fixed

File: Taskfile.full.yml (hunk 2 above)
Comment (ydmeta) [reply Taskfile.full.yml:133]: thanks, fixed

File: Taskfile.full.yml (hunk 3 above)
Comment (ydmeta) [reply Taskfile.full.yml:163]: not in use, removed, nice catch

**Review by ydmeta — COMMENTED** [review]
task test is working, still need add work on venv resolve

**Pull request merged**